### PR TITLE
Adds cleanupsyncs command parameters

### LINKED ISSUE #184: Add parameters for sync filter and instance (server or client) to cleanupsyncs
## Summary
The `cleanupsyncs` [command](https://github.com/learningequality/morango/blob/release-v0.7.x/morango/management/commands/cleanupsyncs.py) is helpful for reducing bloat caused by incomplete sync buffers. Although, since it must prevent the sync from being resumed, it puts the transfer sessions into a state that isn't retroactively helpful for observability.

## Deliverables 
The `cleanupsyncs` command should:
- support a `sync_filter` parameter, which filters to only those transfer sessions with a matching sync filter
- support a `client_instance_id` parameter, which filters to only those sync sessions having a matching `client_instance_id`
- support a `server_instance_id` parameter, which filters to only those sync sessions having a matching `server_instance_id`
- support `push` and `pull` boolean parameters, which filters the transfer sessions on [`push`](https://github.com/learningequality/morango/blob/release-v0.7.x/morango/models/core.py#L275)
- mark the sync session as inactive if all related transfer sessions have been marked inactive, either by successful sync or through the command's processing

### PR DESCRIPTION
**Pull request by akolson**
## Summary

This PR adds parameters for sync filter and instance (server or client) to `cleanupsyncs` parameters added include;
- `sync_filter `
- `client_instance_id `
- `server_instance_id `
- `push` and `pull`

## TODO

- [x] Have tests been written for the new code?
- [ ] Has documentation been written/updated?
- [ ] New dependencies (if any) added to requirements file

## Reviewer guidance

*If you PR has a significant size, give the reviewer some helpful remarks*

## Issues addressed
Fixes #184 

List the issues solved or partly solved by the PR

## Documentation

If the PR has documentation, link the file here (either .rst in your repo or if built on Read The Docs)

### CODE REVIEW

**Review by bjester — CHANGES_REQUESTED** [review]
Looks great! Although, I believe I misled you about the dashes and underscores (messaged on Slack too). 

Please define the arguments with `-` (dashes) instead of `_` (underscores).

**Comment by akolson** [comment]
Changes have been implemented as requested by reviewer.

**Review by bjester — APPROVED** [review]
Approved! Tests pass locally and we'll make a 0.6.x release then merge up to 0.7.x

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Release Notes
 
 List of the most important changes for each release.
+## 0.16.17
```
Comment (bjester) [inline CHANGELOG.md:4]: Oh oops! I just noticed this is `0.16...` not `0.6` :smile:

Comment (akolson) [reply CHANGELOG.md:4]: Let me make change

Comment (akolson) [reply CHANGELOG.md:4]: 0.16 effect 😄

**Pull request merged**